src/index.js

```diff
@@ -23,82 +27,108 @@ module.exports = new BaseKonnector(start)
 
 async function start(fields) {
   await this.deactivateAutoSuccessfulLogin()
-  await authenticate(fields.login, fields.password)
+  await authenticate(fields.identifiant, fields.secret)
```

Comment:
Be careful, do not change fields attributes like this. Since the manifest is not changed, it won't work and you will have some migration problème for the users which already have an ensap account configured with the previous fields keys.

Reply:
Ok I'll change that.